package.json

```diff
@@ -54,11 +56,13 @@
   },
   "_comment": "We don't include @theforeman/vendor because it's assumed to be present in Foreman",
   "dependencies": {
+    "@patternfly/react-icons": "^3.15.15",
```

Comment:
This needs to be added to @theforeman/vendor, I realized this today. Will open up  a PR and link it here :+1:

Reply:
Added here https://github.com/theforeman/foreman-js/pull/133